src/esp/assets/ResourceManager.cpp

```diff
@@ -1207,5 +1207,49 @@ bool ResourceManager::loadSUNCGHouseFile(const AssetInfo& houseInfo,
   return true;
 }
 
+vec3f toEig(Magnum::Vector3 v) {
```

Comment:
This can be done via `Mn::EigenIntegration::cast<vec3f>`

Reply:
Thanks, hoping someone would have a shortcut for this. :+1: